plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -133,6 +134,7 @@ export default function Content(): JSX.Element {
 
 	return (
 		<div className="woocommerce-marketplace__content">
+			<InstallNewProductModal products={ products } />
```

Comment:
Currently, this flow doesn't work on the Featured/Discover page. Because there is no `Product[]` in the top level. I need to modify the data (it comes in groups and not a flat list), lift it up to `<Content>` and pass it down to the `InstallNewProductModal`. This is going to be a separate PR.

Reply:
I would leave this as optional in case we don't have time.

Reply:
OK, thanks!